_posts/2020-10-16-kaitai-struct-v0.9-released.md

```diff
@@ -13,41 +13,42 @@ categories: news
 can be compiled using `kaitai-struct-compiler` into source files in
 one of the supported programming languages. These modules will include
 a generated code for a parser that can read described data structure
-from a file / stream and provide access to its contents in a nice,
+from a file / buffer and provide access to its contents in a nice,
 easy-to-comprehend API.
 
 With the previous 0.8 release, Kaitai project celebrated 1000 stars on GitHub,
 and until 0.9 version, it has collected more than [2000 stars](https://github.com/kaitai-io/kaitai_struct/stargazers).
 Thank you all for your support!
 
 This version introduces C++11 target (which uses smart pointers),
-several handy features (like `valid`ations and little-endian bit integers),
-fixes a lot of bugs and includes quite a few infrastructure improvements.
+several handy features (like `valid`ations and little-endian "bit-sized
+types"), fixes a lot of bugs and includes quite a few infrastructure
+improvements.
 
 ## Release highlights
 
 * New targets support:
-  * Python with [Construct](https://construct.readthedocs.io) library
-  * HTML - intended for documentation, preliminary support
-  * Nim - entry-level support (51% tests pass score)
-* New KSY language features:
+  * Python with [Construct](https://construct.readthedocs.io) library ([#377](https://github.com/kaitai-io/kaitai_struct/issues/377))
+  * HTML - intended for documentation, preliminary support ([ec21b](https://github.com/kaitai-io/kaitai_struct_compiler/commit/ec21bdf62bcc9211e514dd8413ed5f6714a8e6bc))
+  * Nim - entry-level support (51% tests pass score) ([#619](https://github.com/kaitai-io/kaitai_struct/issues/619))
+* New KS language features:
   * `doc-ref` supports list of references ([#269](https://github.com/kaitai-io/kaitai_struct/issues/269))
   * `meta/tags` allows specification of multiple tags to allow better navigation in the format gallery ([#572](https://github.com/kaitai-io/kaitai_struct/issues/572))
   * Allow accessing nested types using `::` syntax: `foo::bar` ([#275](https://github.com/kaitai-io/kaitai_struct/issues/275))
   * Implement parsed data validations using `valid` key ([#435](https://github.com/kaitai-io/kaitai_struct/issues/435))
   * Implement compile-time `sizeof` and `bitsizeof` operators ([#84](https://github.com/kaitai-io/kaitai_struct/issues/84))
     * Type-based: `sizeof<u4>`, `bitsizeof<b13>`, `sizeof<user_type>`
     * Value-based: `file_header._sizeof`, `flags._bitsizeof` (`file_header`, `flags` are fields defined in the current type)
-  * Implement little-endian bit-sized integers ([docs](https://doc.kaitai.io/user_guide.html#bit-ints-le))
-    * Support choosing endianness using `le` / `be` suffix: `type: b12le`, `type: b1be`
-    * Add `meta/bit-endian` key for selecting default bit endianness (`le` / `be`)
+  * Implement little-endian-based bit-sized types ([docs](https://doc.kaitai.io/user_guide.html#bit-ints-le))
```

Comment:
What about just using the established terminology from the User Guide (https://doc.kaitai.io/user_guide.html#bit-ints-le) to which you had no serious objections (just ["some offtop"](https://github.com/kaitai-io/kaitai_struct_doc/pull/43#discussion_r490750324))?

```suggestion
  * Implement bit-sized integers in little-endian order ([docs](https://doc.kaitai.io/user_guide.html#bit-ints-le))
```

It would be also OK to use _little-endian layout_ for example.

Yeah, I'm still not a fan of _bit-sized types_.

Reply:
The problem is that that feature is mainly not about integers in little-endian order. Concept of endianness makes sense only for the stuff that have the size of multiple of endianness granularity (8 bit currently) and is aligned with that granularity. And it is not restricted to integers, it is just that bits sequences can be processed as integers and in order to process them on CPUs they have to be treated this way.